deploy/operator/ztp/day2BaremetalHost.j2

```diff
@@ -0,0 +1,38 @@
+{% set ns = namespace(counter=1) %}
+{% set bmhs = [] %}
+{% set bmhs = baremetalhosts %}
+{% for host in bmhs %}
+
+---
+apiVersion: v1
+kind: Secret
+metadata:
+  name: '{{ host["name"] }}-bmc-secret'
+  namespace: '{{ spoke_namespace }}'
+type: Opaque
+data:
+  username: '{{ host["driver_info"]["username"] | b64encode }}'
+  password: '{{ host["driver_info"]["password"] | b64encode }}'
+
+---
+apiVersion: metal3.io/v1alpha1
+kind: BareMetalHost
+metadata:
+  name: '{{ host["name"] }}'
+  namespace: '{{ spoke_namespace }}'
+  labels:
+    infraenvs.agent-install.openshift.io: '{{ infraenv_name }}'
+  annotations:
+    bmac.agent-install.openshift.io/hostname: '{{ host["name"] }}'
+spec:
+  online: true
+  bootMACAddress: '{{ host["ports"][0]["address"] }}'
+  bmc:
+    address: '{{ host["driver_info"]["address"] }}'
+    credentialsName: '{{ host["name"] }}-bmc-secret'
+  image:
+    format: 'live-iso'
+    url: '{{ download_url }}'
```

Comment:
Can you elaborate on why you are adding the `image` section here? BMAC should take care of this

Reply:
I think I needed that.  Checking ...

Reply:
if you need it, then there's a bug. BMAC should add it aumatically

Reply:
You are right. Not needed